src/test/java/io/neonbee/cluster/LocalRequestClusterTest.java

```diff
@@ -26,6 +27,7 @@
 import io.vertx.junit5.VertxTestContext;
 
 @ExtendWith(NeonBeeExtension.class)
+@Isolated("Some of the methods in this test class run clustered and use the FakeClusterManager for it. The FakeClusterManager uses a static state and can therefore not be run with other clustered tests.")
```

Comment:
Is it possible to add the @Isolated to all tests that are based on NeonBeeExtension?

Reply:
Thanks for the review @pk-work. I have checked the JUnit documentation and also some of their code, but I found no apparent way, how that could be feasible unfortunately... good idea though.